scipy/stats/tests/test_stats.py

```diff
@@ -7194,14 +7194,14 @@ def test_3d_inputs(self):
 
     def test_length0_1d_error(self):
         # Require at least one value in each group.
-        msg = 'all input arrays have length 1.'
-        with assert_warns(stats.DegenerateDataWarning, match=msg):
+        msg = 'at least one input has length 0'
```

Comment:
Yes, this is the context in which I noticed the problem. The new message is correct.